pkgs/development/tools/analysis/cutter/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchFromGitHub, fetchpatch, qmake, pkgconfig, qtbase, qtsvg, radare2 }:
+
+
+let tag = "1.1"; in
+stdenv.mkDerivation rec {
+  name = "r2-cutter-${version}";
+  version = "${tag}.0";
```

Comment:
Why do you `.0` to the version number?

Reply:
Hmm, don't remember.  Something worked better with "full" version specification, maybe some tool I use.  If it seems odd I'll change it :).